tests/installation/opensuse_welcome.pm

```diff
@@ -15,7 +15,20 @@ use utils;
 use x11utils 'handle_welcome_screen';
 
 sub run {
-    handle_welcome_screen;
+    my $upgrade = get_var('UPGRADE');
+    if ($upgrade == 1) {
+        send_key 'esc';
+        x11_start_program('xterm');
+        my $config = script_run("ls .config/autostart | grep org.opensuse.opensuse_welcome.desktop");
```

Comment:
Sounds like `test -f` is more appropriate

Reply:
Or both actually, sth like:
`test -f .config/autostart` -> file is there
`ls .config/autostart | grep org.opensuse.opensuse_welcome.desktop` -> org.opensuse.opensuse_welcome.desktop is not mispelled for example

Reply:
I meant `test -f ~/.config/autostart/org.opensuse.opensuse_welcome.desktop`